src/ui/steps/UncompleteStep.h

```diff
@@ -5,13 +5,13 @@
 #ifndef TASKMANAGER_SRC_UI_STEPS_UNCOMPLETESTEP_H_
 #define TASKMANAGER_SRC_UI_STEPS_UNCOMPLETESTEP_H_
 
-#include "Step.h"
+#include "IOStep.h"
 #include "utilities/StepUtils.h"
 #include "ui/actions/UncompleteTaskAction.h"
 
-class UncompleteStep : public Step {
+class UncompleteStep : public IOStep {
 public:
-    using Step::Step;
+    using IOStep::IOStep;
```

Comment:
avoid this notation as a whole. If you use something from parent — make it explicit

Reply:
fixed